src/containers/shared/transactionUtils.ts

```diff
@@ -68,6 +68,10 @@ export const TX_FLAGS: Record<string, Record<number, string>> = {
 }
 
 export const ACCOUNT_FLAGS: Record<number, string> = {
+  15: 'asfDisallowIncomingTrustline',
+  14: 'asfDisallowIncomingPayChan',
+  13: 'asfDisallowIncomingCheck',
+  12: 'asfDisallowIncomingNFTOffer',
```

Comment:
Is there no 11?

Reply:
11 is commented out in the rippled source as being reserved for hooks later.  We can add it when we enhance hook support.  I quickly tried looking but could not find it official referenced in the docs for hooks.

Reply:
```
/*  // reserved for Hooks amendment
    lsfTshCollect = 0x02000000,     // True, allow TSH collect-calls to acc hooks
*/
```